From 0a5e493ea20d9d24d0808d8a4e05a97deb19b423 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Fri, 3 Jul 2020 16:13:33 +0100
Subject: [PATCH] drm/vc4: FKMS: Put includes in alphabetical order,
 and use <> instead of ""

Reorder the includes, and use the system include paths rather than
local ones

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 drivers/gpu/drm/vc4/vc4_firmware_kms.c | 29 +++++++++++++-------------
 1 file changed, 15 insertions(+), 14 deletions(-)

--- a/drivers/gpu/drm/vc4/vc4_firmware_kms.c
+++ b/drivers/gpu/drm/vc4/vc4_firmware_kms.c
@@ -14,25 +14,26 @@
  * Pi's firmware display stack.
  */
 
+#include <drm/drm_atomic_helper.h>
+#include <drm/drm_crtc_helper.h>
+#include <drm/drm_drv.h>
+#include <drm/drm_fb_cma_helper.h>
+#include <drm/drm_fourcc.h>
+#include <drm/drm_gem_framebuffer_helper.h>
+#include <drm/drm_plane_helper.h>
+#include <drm/drm_probe_helper.h>
+#include <drm/drm_vblank.h>
+
+#include <linux/component.h>
+#include <linux/clk.h>
+#include <linux/debugfs.h>
 #include <linux/module.h>
 
-#include "drm/drm_atomic_helper.h"
-#include "drm/drm_gem_framebuffer_helper.h"
-#include "drm/drm_plane_helper.h"
-#include "drm/drm_crtc_helper.h"
-#include "drm/drm_fourcc.h"
-#include "drm/drm_probe_helper.h"
-#include "drm/drm_drv.h"
-#include "drm/drm_vblank.h"
-#include "linux/clk.h"
-#include "linux/debugfs.h"
-#include "drm/drm_fb_cma_helper.h"
-#include "linux/component.h"
-#include "linux/of_device.h"
+#include <soc/bcm2835/raspberrypi-firmware.h>
+
 #include "vc4_drv.h"
 #include "vc4_regs.h"
 #include "vc_image_types.h"
-#include <soc/bcm2835/raspberrypi-firmware.h>
 
 int fkms_max_refresh_rate = 85;
 module_param(fkms_max_refresh_rate, int, 0644);
